docs/services/sharelatex.md

```diff
@@ -63,23 +63,34 @@ ShareLaTeX requires a redis and a mongodb instance!
 ```
 
 ### Installation of texlive-full
-!!! warning ""
+**Attention**
```

Comment:
Can you explain? Checkout [https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage)